shell/platform/android/io/flutter/app/FlutterFragmentActivity.java

```diff
@@ -0,0 +1,145 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.app;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.res.Configuration;
+import android.os.Bundle;
+import android.support.v4.app.FragmentActivity;
+import io.flutter.app.FlutterActivityDelegate.ViewFactory;
+import io.flutter.plugin.common.PluginRegistry;
+import io.flutter.plugin.common.PluginRegistry.Registrar;
+import io.flutter.view.FlutterView;
+
+/**
+ * Base class for activities that use Flutter who also require the use of the
+ * Android v4 Support library's {@link FragmentActivity}. Applications that
+ * don't have this need will likely want to use {@link FlutterActivity} instead.
+ * <p/>
```

Comment:
Yeah, Javadoc isn't as smart about formatting as Dartdoc - with the exception of `{@foo}` directives, it treats everything as raw HTML, as in so doing, it collapses your paragraphs into one unless you add `<br/>` or `<p/>` tags.